lib/Doctrine/Common/Inflector/Inflector.php

```diff
@@ -31,7 +31,7 @@ class Inflector
     /**
      * Plural inflector rules.
      *
-     * @var string[][]
+     * @var string|string[][]
```

Comment:
I think this is definitely wrong. It is array of arrays, not a single string... Why was this done? Did PHPStan suggest that?

Reply:
The error log:
```
 ------ ------------------------------------------------------ 
  Line   lib/Doctrine/Common/Inflector/Inflector.php           
 ------ ------------------------------------------------------ 
  489    Array (array<string>) does not accept array<string>.  
  497    Array (array<array<string>>) does not accept string.  
  498    Array (array<array<string>>) does not accept string.  
  550    Array (array<array<string>>) does not accept string.  
  551    Array (array<array<string>>) does not accept string.  
 ------ ------------------------------------------------------
```

Reply:
Looks like PHPStan has some hiccups here. Might be worth reporting upstream, in the meantime I'd be fine merging this since the members are private. Maybe we can add a comment explaining it?